charm/serial-vault/config.yaml

```diff
@@ -0,0 +1,73 @@
+options:
+    title:
+        description: "service display name"
+        type: string
+        default: "Serial Vault"
+    logo:
+        description: "path to the logo icon"
+        type: string
+        default: "/static/images/logo-ubuntu-white.svg"
+    docRoot:
+        description: "Path to the assets (${docRoot}/static)"
+        type: string
+        default: "."
+    mode:
+        description: "Service mode=signing for API or mode=admin for UI"
+        type: string
+        default: "admin"
+    portAdmin:    
+        description: "Port for the service mode=admin"
+        type: string
+        default: "8081"
+    portSigning:  
+        description: "Port for the service mode=signing"
+        type: string
+        default: "8080"
+    dbdriver:
+        description: "database driver"
+        type: string
+        default: "postgres"
+    keystore:
+        description: "Signing key store, can be: databse, filesystem, memory"
```

Comment:
databse -> database (it's the default so it's not a huge deal/typo)

Reply:
fixed